src/EnergyPlus/SingleDuct.cc

```diff
@@ -6015,6 +6015,26 @@ namespace SingleDuct {
         }
     }
 
+    void setATMixerSizingProperties(int const &inletATMixerIndex, int const &controlledZoneNum, int const &curZoneEqNum)
+    {
+        ZoneEqSizing(curZoneEqNum).ATMixerVolFlow = SingleDuct::SysATMixer(inletATMixerIndex).DesignPrimaryAirVolRate;
```

Comment:
Maybe I should move this air vol flow down into if (SizingDesRunThisAirSys) so that all are set or none are set?